modules/reference/pages/feature/transportSecurity/description.adoc

```diff
@@ -0,0 +1,4 @@
+When the Transport security feature is enabled, the SSL context for the process is the default SSL context of the Java Secure Socket Extension (JSSE).
```

Comment:
sorry I missed this before- feature names should get headline-style capitalization

Transport security feature
--->
Transport Security feature